api/src/test/java/org/pmiops/workbench/cdr/CdrVersionServiceTest.java

```diff
@@ -5,12 +5,10 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.Mockito.when;
 
-import com.google.common.collect.ImmutableList;
```

Comment:
the only meaningful change to this file is updating `assertResponseMultiTier()` and renaming it to `assertExpectedResponse()` - the rest is just cleanup